components/storage/ComparisonCards.vue

```diff
@@ -16,19 +16,18 @@
       class="mx-3 my-3 px-3"
     >
       <template #header>
-        <h2 class="title is-size-4">{{ service.service | humanize }}</h2>
+        <h2 class="title is-size-4">{{ service.service }}</h2>
```

Comment:
```suggestion
        <h2 class="title is-size-4">{{ humanize(service.service) }}</h2>
```
unless no longer needed